test/resources/application.conf

```diff
@@ -0,0 +1,19 @@
+# https://www.playframework.com/documentation/latest/Configuration
+play.http.port=9000
+
+auth.url="http://localhost:9005"
+auth.callback="http://localhost:9000/callback"
+auth.secret="f5d12779-2927-4992-a7cf-77358cf42e20"
+auth.secret=${?AUTH_SECRET}
```

Comment:
Is this line necessary in the test config?

Reply:
No it isn't. That's lazy copy and pasting by me :smile: